pkg/proxy/engines/proxy_request.go

```diff
@@ -237,8 +266,9 @@ func (pr *proxyRequest) prepareUpstreamRequests() {
 		pr.originRequests = make([]*http.Request, 0, l)
 	}
 
-	// TODO: This looks suspicious
-	rsc.OriginConfig.DearticulateUpstreamRanges = true
+	// // TODO: This looks suspicious
+	// rsc.OriginConfig.DearticulateUpstreamRanges = true
```

Comment:
?

Reply:
I believe this was put in for debugging purposes and forgotten about; so it is commented for Beta2 and will be scrubbed completely in the next release